amd/src/input.js

```diff
@@ -19,64 +19,62 @@
  * @author     André Storhaug <andr3.storhaug@gmail.com>
  * @copyright  2018 NTNU
  */
-
+import VisualMath from "???";
+import Ajax from "core/ajax";
+import notification from "???";
```

Comment:
How do I even import core modules? Relatively, the core modules exist at "../../../../amd/src/<script>.js", but there has to be a better way?

Reply:
@andstor

Reply:
The core modules can be accessed (imported) by writing 
```js
import xxx from "core/moduleName";
```

So the notifications module is available at `core/notification`.

However, `VisualMath` is local to the `ShortMath` repo, so you would probably need to reference it relatively.